doc/changes.rst

```diff
@@ -0,0 +1,95 @@
+Changelog
+=========
+
+Release v0.1.0
+--------------
+
+*Released on: 2020/04/XX*
```

Comment:
```suggestion
Release v0.1.0 (2020/04/XX)
---------------------------
```

Would this look better?

Reply:
Ok, cutting the date to 30 April 2020! Using Greenwich Meridian Time (the other GMT).